README.md

```diff
@@ -1,10 +1,10 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task-move-spider-DOM/)
+    - [DEMO LINK](https://viktoriyaRi.github.io/js_task-move-spider-DOM/)
```

Comment:
Add this link to the PR description 
<img width="941" alt="image" src="https://github.com/mate-academy/js_task-move-spider-DOM/assets/64750958/e729e83c-f1d6-4007-94de-a3c779694f96">